packages/cli/src/lib/bundler/types.ts

```diff
@@ -22,6 +22,7 @@ export type BundlingOptions = {
   isDev: boolean;
   config: Config;
   appConfigs: AppConfig[];
+  isBackend?: boolean;
```

Comment:
yeah +1 to keeping all of theme mandatory, since it's just internal api and we want it to be as readable as possible, and we don't really care too much about sane defaults.